vendor/bat-native-ads/src/bat/ads/internal/notifications.cc

```diff
@@ -49,76 +49,91 @@ void Notifications::Initialize(InitializeCallback callback) {
 bool Notifications::Get(const std::string& id, NotificationInfo* info) const {
   DCHECK(is_initialized_);
 
-  auto notification = notifications_.find(id);
-  if (notification == notifications_.end()) {
+  auto iter = std::find_if(notifications_.begin(), notifications_.end(),
```

Comment:
Please be consistent with other lambda code style in ads

Reply:
@tmancey what exactly is inconsistent with lambdas code style. Please show an example.

Reply:
This was a nit, so resolving